lib/screens/app/explorer/proposals/proposal_details.dart

```diff
@@ -120,7 +120,7 @@ class ProposalDetailsPageState extends State<ProposalDetailsPage> {
             ),
             SizedBox(height: 8),
             Text(
-              'Requested amount: ${amountFormatter.output.nonSymbol} SEEDS',
+              'Requested amount: $quantity SEEDS',
               style: textTheme.subhead,
```

Comment:
The quantity that was returning from the server was in the form of "5.22 SEEDS"
did you change this behavior?

Reply:
It's always returned from blockchain with 4 decimals ("5.2200 SEEDS")

Reply:
In this case, I had to remove money formatter package because it was not compatible with some another package...

Reply:
then remove the seeds word after the $quantity
because then you would get "5.2200 SEEDS SEEDS"

Reply:
quantity is a double... so should be formatted with min 2 and max 4 decimals, not plain double value

also this
double.tryParse(proposal.quantity.replaceAll(RegExp(r' SEEDS'), '')) ??
             0.0;

why is it using 0.0? it should throw an error.